test/flows/check_travel_during_coronavirus_flow_test.rb

```diff
@@ -352,11 +352,11 @@ class CheckTravelDuringCoronavirusFlowTest < ActiveSupport::TestCase
 
       should "render guidance for people who aren't fully vaccinated" do
         add_responses vaccination_status: "9ddc7655bfd0d477"
-        assert_rendered_outcome text: "people who aren't fully vaccinated"
+        assert_rendered_outcome text: "Returning to England if you're not fully vaccinated"
```

Comment:
We should probably think about internationalising this at some point to avoid having to do things like this

Reply:
Agree. However, and perhaps surprisingly - at the moment the app is [not I18N enabled](https://github.com/alphagov/smart-answers/blob/main/config/locales/en.yml). I suspect to do so would be quite a large task.